packages/wells/src/client/api/wells.ts

```diff
@@ -40,10 +47,26 @@ export class Wells extends AssetsAPI {
         id: asset.id,
         name: asset.name,
         wellHeadLocation: Wells.mapToWellHeadLocation(asset),
+        dataSource: Wells.getLabel(DATA_SOURCE, asset.labels),
```

Comment:
Just one more question - does getLabel return an optional? We need to consider the possibhility that people may start with missing data (not having some of the info filled yet). If this returns an optional label and then the attribute can be set to null, should be ok, but just wanted to check ;)